explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -356,10 +359,13 @@ object TargetTabContents {
         agwo.copy(asterismGroups = updatedAsterismGroups, targetGroups = updatedTargetGroups)
       }
 
-      val asterismView: ReuseView[List[TargetWithId]] = asterismGroupWithObs.map(
-        _.withOnMod(onModAsterismsWithObs(groupIds, idsToEdit))
-          .zoom(getAsterism)(modAsterism)
-      )
+      val asterismView: ReuseView[List[TargetWithId]] =
+        asterismGroupsWithObs
+          .map(
+            _.withOnMod(onModAsterismsWithObs(groupIds, idsToEdit))
+              .zoom(getAsterism)(modAsterism)
+          )
+          .addReuseByFrom(panels)
```

Comment:
This is the actual fix. It adds the selected panel information to reusability.

The rest is format/renaming.